services/quests/errors/errors.test.tsx

```diff
@@ -38,7 +38,7 @@ describe('Errors', () => {
         // Note the requirement for only one error. Error invalid test cases should be designed
         // such that they don't trigger multiple errors, so as to prevent confusion.
         const errorName = (err.INVALID_ERRORS && err.INVALID_ERRORS[index] !== null) ? err.INVALID_ERRORS[index] : err.NAME;
-        expect(msgs.error.length).toEqual(1, `Length !== 1, was ${msgs.error.length}: ${msgs.error.map((e) => e.text).join('...')}`);
+        expect(msgs.error.length).toEqual(1); // `Length !== 1, was ${msgs.error.length}: ${msgs.error.map((e) => e.text).join('...')}`
```

Comment:
Change this to `assert.equal` so we can preserve the debug text.

Context: https://github.com/facebook/jest/issues/1965#issuecomment-354423445